### ISSUE TITLE
silence text output after import of gaia module

### ISSUE THREAD

**Issue by jpdeleon** [astropy/astroquery#1616]
How to silence the output: 
```
Created TAP+ (v1.2.1) - Connection:
	Host: gea.esac.esa.int
	Use HTTPS: True
	Port: 443
	SSL Port: 443
```
after importing gaia via `from astroquery.gaia import Gaia`?

**Comment by keflavich** [comment]
This will likely have to wait on an upcoming Gaia refactor to use pyvo.

**Comment by eerovaher** [comment]
The output described here is silenced by default since #1722. This issue is fixed and can be closed.

**Comment by keflavich** [comment]
Closed by #2085
